src/mantarray_waveform_analysis/peak_detection.py

```diff
@@ -352,6 +345,46 @@ def find_twitch_indices(
     return twitches
 
 
+def _too_few_peaks_or_valleys(
+    peak_indices: NDArray[int], valley_indices: NDArray[int]
+) -> None:
+    """Raise an error if there are too few peaks or valleys detected.
+
+    Args:
+        peak_indices: a 1D array of integers representing the indices of the peaks
+        valley_indices: a 1D array of integeres representing the indices of the valleys
+    """
+    if len(peak_indices) < MIN_NUMBER_PEAKS:
+        raise TooFewPeaksDetectedError(
+            f"A minimum of {MIN_NUMBER_PEAKS} peaks is required to extract twitch metrics, however only {len(peak_indices)} peak(s) were detected"
+        )
+    if len(valley_indices) < MIN_NUMBER_VALLEYS:
+        raise TooFewPeaksDetectedError(
+            f"A minimum of {MIN_NUMBER_VALLEYS} valleys is required to extract twitch metrics, however only {len(valley_indices)} valley(s) were detected"
+        )
+
+
+def _find_start_indices(starts_with_peak: bool) -> Tuple[int, int]:
+    """Find start indices for peaks and valleys.
+
+    Args:
+        peak_indices: a 1D array of integers representing the indices of the peaks
```

Comment:
please update the `Args` docstring to match the function.  I'm wondering if there's an automated tool that we could use that checks whether it is out of sync with the function signature.  I would have thought `pylint` would do that, or maybe `docformatter` but I guess not